examples/interaction/tools/tap_inspect.py

```diff
@@ -0,0 +1,52 @@
+import numpy as np
```

Comment:
Maybe `tap_inspect_keymod.py` just to highlight that this example covers key modifiers?